platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/LibraryLoader.java

```diff
@@ -16,12 +16,17 @@
   private static final LibraryLoader DEFAULT = new LibraryLoader() {
     @Override
     public void load(String name) {
-      System.loadLibrary(name);
+      if (!loaded) {
+        System.loadLibrary(name);
+      }
+      loaded = true;
```

Comment:
Can we move the point of setting this flag (and the check) to the static `load` method so that it's set for custom library loaders as well?

Reply:
For this I exposed the loaded field as protected, I didn't want to enforce logic on the user but on the other end not seeing a use-case for not checking the flag for all loaders.